include/picolibrary/microchip/megaavr0/peripheral.h

```diff
@@ -38,6 +39,42 @@ namespace picolibrary::Microchip::megaAVR0::Peripheral {
  */
 using CLKCTRL0 = Instance<CLKCTRL, 0x0060>;
 
+/**
```

Comment:
VPORT instances should appear before CLKCTRL0 since they are at lower addresses.